packages/lumx-core/src/scss/core/state/_mixins.scss

```diff
@@ -107,7 +107,7 @@
 }
 
 @mixin lumx-state-disabled-input() {
-    pointer-events: none;
+    cursor: not-allowed;
```

Comment:
`pointer-events` must be kept otherwise you'll enable mouse event on all disabled elements that use this mixin (Some of those elements might not have a native `disabled` attribute that also disables all interactions).
 
```suggestion
    cursor: not-allowed;
    pointer-events: none;
```

Reply:
Hmm 🤔. So we have 2 solution

1. Remove pointer-events and make sure we don't have regression on:
    packages/lumx-core/src/scss/components/button
    packages/lumx-core/src/scss/components/checkbox
    packages/lumx-core/src/scss/components/chip
    packages/lumx-core/src/scss/components/link
    packages/lumx-core/src/scss/components/list
    packages/lumx-core/src/scss/components/radio-button
    packages/lumx-core/src/scss/components/select
    packages/lumx-core/src/scss/components/switch
    packages/lumx-core/src/scss/components/text-field
    And in `lumapps-web`.
    I tested on buttons, it looks like removing `pointer-event` will make the `hover` state appear in `disabled` were this was not present before 🤷 
2. Apply the cursor only on buttons like you said (but designers will probably ask for this cursor on all disabled components)